Sources_v3/Controllers/MemberController/MemberController.swift

```diff
@@ -0,0 +1,339 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+import Foundation
+
+public extension _ChatClient {
+    /// Creates a new `_ChatMemberController` for the user with the provided `userId` and `cid`.
+    /// - Parameters:
+    ///   - userId: The user identifier.
+    ///   - cid: The channel identifier.
+    /// - Returns: A new instance of `_ChatMemberController`.
+    func memberController(userId: UserId, in cid: ChannelId) -> _ChatMemberController<ExtraData> {
+        .init(userId: userId, cid: cid, client: self)
+    }
+}
+
+/// `ChatMemberController` is a controller class which allows mutating and observing changes of a specific chat member.
+///
+/// `ChatMemberController` objects are lightweight, and they can be used for both, continuous data change observations,
+/// and for quick user actions (like ban/unban).
+///
+/// - Note: `ChatMemberController` is a typealias of `_ChatMemberController` with default extra data. If you're using custom
+/// extra data, create your own typealias of `_ChatMemberController`.
+///
+/// Learn more about using custom extra data in our [cheat sheet](https://github.com/GetStream/stream-chat-swift/wiki/StreamChat-SDK-Cheat-Sheet#working-with-extra-data).
+///
+public typealias ChatMemberController = _ChatMemberController<DefaultExtraData>
```

Comment:
Should this be called `ChatChannelMemberController` ? cc @b-onc

Reply:
fixed in 973619a02a94368cdd60541b66007f8e1ec5a3fe and also renamed `BanUserRequestPayload -> ChannelMemberBanRequestPayload`